comb_spec_searcher/specification.py

```diff
@@ -69,6 +69,18 @@ def __init__(
         if group_equiv:
             self._group_equiv_in_path()
         self._set_subrules()
+        self._enforce_labels()
+
+    def _enforce_labels(self) -> None:
```

Comment:
Can we add a doctring?

Reply:
Yes, good catch. I'm surprised pylint didn't complain.